docs/how-to/how-to-pull-images-in-guest-with-kata.md

```diff
@@ -178,135 +178,217 @@ Currently, the image pulled in the guest will be downloaded and unpacked in the
+   Units: sectors of 1 * 512 = 512 bytes
+   Sector size (logical/physical): 512 bytes / 512 bytes
+   I/O size (minimum/optimal): 512 bytes / 512 bytes
+   ```
+
+4. Export environment variables
+
+   ```bash
+   export NODE_NAME=$(kubectl get nodes -o jsonpath='{.items[0].metadata.name}')
+   export STORAGE_SIZE="10Gi"
+   export LOOP_DEVICE=$device
+   ```
+
+5. Create file `local-storage-class.yaml` to define Storage Class for the block device
+   ```yaml
+   apiVersion: storage.k8s.io/v1
+   kind: StorageClass
+   metadata:
+     name: local-storage
+   provisioner: kubernetes.io/no-provisioner
+   volumeBindingMode: WaitForFirstConsumer
+   ```
+
+6. Deploy
+
+    ```bash
+    kubectl apply -f local-storage-class.yaml
+    ```
+
+7. Create `trusted-storage-pv.yaml` file to define Persistent Volume (PV) for the block device
+   ```yaml
+   ---
+   apiVersion: v1
+   kind: PersistentVolume
+   metadata:
+     name: trusted-block-pv
+   spec:
+     capacity:
+       storage: ${STORAGE_SIZE}
+     volumeMode: Block
+     accessModes:
+       - ReadWriteOnce
+     persistentVolumeReclaimPolicy: Retain
+     storageClassName: local-storage
+     local:
+       path: ${LOOP_DEVICE}
+     nodeAffinity:
+       required:
+         nodeSelectorTerms:
+           - matchExpressions:
+               - key: kubernetes.io/hostname
+                 operator: In
+                 values:
+                   - ${NODE_NAME}
+   ```
+
+8. Deploy:
+
+   ```bash
+   envsubst < trusted-storage-pv.yaml | kubectl apply -f -
+   ```
+
+9. Create `trusted-storage-pvc.yaml` file to define Persistent Volume Claim (PVC) for the block device
+   ```yaml
+   ---
+   apiVersion: v1
+   kind: PersistentVolumeClaim
+   metadata:
+     name: trusted-pvc
+   spec:
+     accessModes:
+       - ReadWriteOnce
+     resources:
+       requests:
+         storage: ${STORAGE_SIZE}
+     volumeMode: Block
+     storageClassName: local-storage
+   ```
+
+10. Deploy:
+
+    ```bash
+    envsubst < trusted-storage-pvc.yaml | kubectl apply -f -
+    ```
+
+11. Create a file `trusted-storage-pod.yaml` with pulling large image in guest
+
+    ```yaml
+    ---
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: large-image-pod
+    spec:
+      runtimeClassName: kata-qemu
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: kubernetes.io/hostname
+                    operator: In
+                    values:
+                      - ${NODE_NAME}
+      volumes:
+        - name: trusted-storage
+          persistentVolumeClaim:
+            claimName: trusted-pvc
+      containers:
+        - name: app-container
+          image: quay.io/confidential-containers/test-images:largeimage
+          command: ["/bin/sh", "-c"]
+          args:
+            - sleep 6000
+          volumeDevices:
+            - devicePath: /dev/trusted_store
```

Comment:
Is`/dev/trusted_store` a pre-defined device path here in CoCo ? Some comments for this path might be helpful for users to understand what and why.

Reply:
To be honest, I don't know how kata is matching the device for image pull. Tried to find it in the kata documentation but I failed. If any other reviewer can jump in here to explain I'll be glad to add this information to the PR.